src/components/shared/card/Card.tsx

```diff
@@ -0,0 +1,33 @@
+import { BsDot } from "react-icons/bs";
+import { Link, useNavigate } from "react-router-dom";
+import { ROUTES } from "../../../router/routes";
+
+export type CardProps = {
+  id: string;
+  titleName: string;
+  description: string;
+  imgSrc: string;
+};
+
+export function Card({ titleName, description, imgSrc, id }: CardProps) {
+  const navigate = useNavigate();
+
+  return (
+    <Link
+      to={`${ROUTES.doctorDetail.replace(":doctorId", id)}`}
```

Comment:
Super pomysł, ja bym to jedynie przeniósł przed returna i dał const detailsURL = ROUTES... itd, w JSX najlepiej umieszczać głównie zmienne bo są czytelniejsze, zmniejszaja chaos wprowadzany mnogoscia konkstrukcji, zagniezdzen etc